src/Oro/Bundle/ConfigBundle/Resources/config/system_configuration.yml

```diff
@@ -7,6 +7,9 @@ oro_system_configuration:
         general_setup:
             title: oro.config.system_configuration.groups.general_setup.title
 
+        integration_setup:
```

Comment:
Why did you add this group here? I think it should be in  src/Oro/Bundle/UserBundle/Resources/config/system_configuration.yml